app/schemas/components/email.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require('joi')
+
+const { TEMPORARY, PERMANENT } = require('../../constants/notify-simulation-email-addresses')
+
+module.exports = Joi.alternatives().try(
+  Joi.string().email(),
+  Joi.string().valid(TEMPORARY, PERMANENT)
+).required()
+  .messages({
+    'string.empty': 'Email cannot be empty.',
+    'string.email': 'The email provided is invalid.',
+    'alternatives.match': 'Email cannot be empty.',
+    'alternatives.types': 'Email must be a string.',
+    'any.required': 'Email must be provided.',
+    '*': 'The email provided is invalid.'
+  })
```

Comment:
Discrepancies on `Joi` validation for semi-complex things like this.

For `''`
Locally triggers `string.empty` as is said in tests
Joi Sandpit can't really be tested as it doesn't allow `.email`
SND triggers `alternatives.match`

Need to document this on our Confluence page